package.json

```diff
@@ -89,6 +89,7 @@
     "lost": "^6.7.2",
     "moment": "^2.12.0",
     "postcss-import": "^8.0.2",
-    "rucksack-css": "^0.8.5"
+    "rucksack-css": "^0.8.5",
+    "statinamic": "0.9.3"
```

Comment:
@thangngoc89 trying to fix the travis failure, but this seems unsuccessful. anyone thoughts?